demos/src/baz.rs

```diff
@@ -30,42 +38,43 @@ impl Baz {
         std::thread::sleep(delay);
     }
 
-    #[measure(type = HitCount<metered::atomic::AtomicInt<u128>>)]
+    #[measure(type = HitCount<metered::atomic::AtomicInt<u64>>)]
     #[measure(ErrorCount)]
     #[measure(ResponseTime)]
-    pub fn bar(&self, should_fail: bool) -> Result<(), &'static str> {
+    #[measure(BazErrorMetrics)]
+    pub fn bar(&self, should_fail: bool) -> Result<(), BazError> {
```

Comment:
Can you keep the old example to demonstrate `ErrorCount` with the old behavior, and have a separate new example?  I would like to keep backwards compatibility here.